service/updaterconf/updaterconf.go

```diff
@@ -196,6 +195,30 @@ func (u *UpdaterConf) DisableAutoUpdate() error {
 	return nil
 }
 
+func (u *UpdaterConf) removeUpdaterTasks(conf map[string]interface{}, key string) bool {
+	tasks, ok := conf[key].([]interface{})
+	if !ok {
+		return false
+	}
+	for i := 0; i < len(tasks); i++ {
+		task, ok := tasks[i].(map[string]interface{})
+		if !ok {
+			continue
+		}
+		path, ok := task[pathKey].(string)
+		if !ok {
+			continue
+		}
+		if strings.Contains(path, u.updaterFilename) {
```

Comment:
Is updaterFilename configurable somewhere? I may have missed it.

Reply:
yes part of call to
```
 func Create(
	silverDir string,
	silverConfigFilename string,
	updaterFilename string,
)
```